crates/topos-tce-broadcast/src/constant.rs

```diff
@@ -0,0 +1,21 @@
+use lazy_static::lazy_static;
+
+lazy_static! {
+    pub static ref COMMAND_CHANNEL_SIZE: usize =
```

Comment:
doc comments here as well?

Reply:
Sometimes these doc comments seem to obvious, but if you haven't looked at a particular code in  a while, it's sometimes easier to get up to speed by reading the reasoning behind something, at least for me. Reduces the cognitive load a bit.